hypervisor/mmu.c

```diff
@@ -0,0 +1,31 @@
+#include "common.h"
+#define MAX_PAGE_PDE    8
+unsigned int page_directory[1024] __attribute__((aligned(4096)));
+unsigned int page_tables[MAX_PAGE_PDE][1024] __attribute__((aligned(4096)));
+
+
+void init_page_directory() {
+    int i;
+    int j;
+    for(i = 0; i < MAX_PAGE_PDE; i++)
+    {
+        // This sets the following flags to the pages:
+        //   Supervisor: Only kernel-mode can access them
+        //   Write Enabled: It can be both read from and written to
+        //   Not Present: The page table is not present
+        page_directory[i] = 0x00000002;
```

Comment:
i suppose this is the duplicate code with line 26?

Reply:
No. this is the default value.

Reply:
you set this default value here and set another value in line 26 within one function, what's the point of default value?

Reply:
after discussion, for(i = 0; i < MAX_PAGE_PDE; i++) should be 
 for(i = 0; i < 1024; i++)